.ghci

```diff
@@ -1,4 +1,5 @@
 :set -i.:config:dist/build/autogen
+:set -DDEVELOPMENT
```

Comment:
Now that we can `ghci FooSpec.hs` to run individual specs, this is important to prevent an exception if `getEnv` looks for `DATABASE_URL` in non-development.